src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function strNum(str) {
+  const result = [];
+
+  for (let i = 1; i < str.length; i++) {
+    if (Number(str[i]) || str[i] === '0') {
+      result.push(str[i]);
+    }
+  }
+
+  return Number(result.join(''));
```

Comment:
The `strNum` function is not correctly converting the string to a number. It currently extracts digits from the string but does not handle cases where the string might contain non-numeric characters at the start. Consider using `parseFloat` or `parseInt` to convert the entire string to a number.